walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,64 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeAndOptionalAttendeeIsConsidered() {
+    // Have each required attendee have different events. An optional attendee has a whole-day event.
```

Comment:
I really like these diagrams you've added here. This makes understanding the test case crystal clear.